#! /bin/sh /usr/share/dpatch/dpatch-run
## 19include_term_h.dpatch by  <hesso@pool.math.tu-berlin.de>
##
## DP: Add <term.h> to the include list to suppress a few warnings.

@DPATCH@
diff -Naur nvi-1.81.6.orig/cl/cl_funcs.c nvi-1.81.6/cl/cl_funcs.c
--- nvi-1.81.6.orig/cl/cl_funcs.c	2007-11-18 17:41:42.000000000 +0100
+++ nvi-1.81.6/cl/cl_funcs.c	2008-05-01 18:23:08.000000000 +0200
@@ -18,6 +18,7 @@
 #include <sys/time.h>
 
 #include <bitstring.h>
+#include <term.h>
 #include <ctype.h>
 #include <signal.h>
 #include <stdio.h>
diff -Naur nvi-1.81.6.orig/cl/cl_screen.c nvi-1.81.6/cl/cl_screen.c
--- nvi-1.81.6.orig/cl/cl_screen.c	2007-11-18 17:41:42.000000000 +0100
+++ nvi-1.81.6/cl/cl_screen.c	2008-05-01 18:23:02.000000000 +0200
@@ -17,6 +17,7 @@
 #include <sys/queue.h>
 
 #include <bitstring.h>
+#include <term.h>
 #include <errno.h>
 #include <signal.h>
 #include <stdio.h>
